packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -576,22 +580,47 @@ module.exports = (
         await db.createEmail(uid, emailData);
 
         const geoData = request.app.geo;
+        const code = otpUtils.generateOtpCode(hex, otpOptions);
```

Comment:
`code` is only used by the `email-otp` branch, should we just calculate it within that branch and void spinning CPU cycles for the default?